apps/balancer-tools/src/app/apr/round/(components)/MoreFiltersButton.tsx

```diff
@@ -0,0 +1,266 @@
+            sideOffset={5}
+          >
+            <div className="p-2 flex overflow-y-scroll rounded border-[1px] border-blue6 bg-blue3 scrollbar-thin scrollbar-track-blue2 scrollbar-thumb-slate12 w-60">
+              <Accordion.Root className="w-full" type="single" collapsible>
+                {filters.map(({ name, label, options }) => (
+                  <AccordionItem value={label}>
+                    <AccordionTrigger>{label}</AccordionTrigger>
+                    <AccordionContent>
+                      <div className="ml-2 flex flex-col">
+                        {options.map((option) => (
+                          <div key={option} className="flex items-center">
+                            <Checkbox
+                              id={`${name}-${option}`}
+                              checked={
+                                !!selectedAttributes[name]?.includes(option)
+                              }
+                              onChange={() =>
+                                handleAttributeChange(name, option)
+                              }
+                              label={option}
+                            />
+                          </div>
+                        ))}
+                      </div>
+                    </AccordionContent>
+                  </AccordionItem>
+                ))}
+                <AccordionItemMinMax
+                  name="Tvl"
+                  label="TVL"
+                  onChange={handleOnMinMaxChange}
+                  selectedAttributes={selectedAttributes}
+                />
+                <AccordionItemMinMax
+                  name="Apr"
+                  label="APR"
+                  onChange={handleOnMinMaxChange}
+                  selectedAttributes={selectedAttributes}
+                />
+                <AccordionItemMinMax
+                  name="VotingShare"
+                  label="Voting Share (%)"
+                  onChange={handleOnMinMaxChange}
+                  selectedAttributes={selectedAttributes}
+                />
+              </Accordion.Root>
+            </div>
+          </Popover.Content>
+        </Popover.Portal>
+      </Popover.Root>
+    </>
+  );
+}
+
+export default function AccordionItemMinMax({
+  name,
+  label,
+  onChange,
+  selectedAttributes,
+}: {
+  name: keyof SelectedAttributesType;
+  label: string;
+  onChange: (inputName: string, value: string) => void;
+  selectedAttributes: SelectedAttributesType;
+}) {
+  return (
+    <AccordionItem value={String(name)}>
+      <AccordionTrigger>{label}</AccordionTrigger>
+      <AccordionContent>
+        <div className="py-2 flex flex-col">
+          <div className="flex items-center gap-1">
+            <BaseInput
+              className="flex-1"
+              name={`min${name}`}
+              placeholder="Minimum"
+              onChange={(e: ChangeEvent<HTMLInputElement>) =>
+                onChange(`min${name}`, e.target.value)
```

Comment:
`e.target.name` is probably the same as `min${name}`, no? then you could also change your onChange fn